feed/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.css">
```

Comment:
```suggestion
```

This is not needed, bootstrap should be in your dist file.

Reply:
Could you elaborate? I don't see Bootsrap in my dist folder?